net-misc/turbovnc/turbovnc-2.2.5-r2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake
+
+DESCRIPTION="A fast replacement for TigerVNC"
+HOMEPAGE="https://www.turbovnc.org/"
```

Comment:
This is a complete nit and not necessary, but we should add the GitHub site to metadata.xml as a remote-id.